content/en/docs/concepts/containers/container-environment.md

```diff
@@ -2,7 +2,7 @@
 reviewers:
 - mikedanese
 - thockin
-title: Container Environment Variables
+title: Container Environment
```

Comment:
Hello @rajeshdeshpande02 , A couple of thoughts:
With this change, the title of the page is identical to the main heading text, `Container Environment`.
I think the title generally refers to Kubernetes container environment variables (for configuring containers).

If you choose to update the title (the _redirects file may need an update too), you should probably change the h2 level heading to something like, `## Container environment variables`.

Reply:
Changes seem okay. I don't think see much difference between the original text and the proposed text.
/lgtm